README.md

```diff
@@ -65,5 +65,55 @@ that isnt available is annoying. Let alone it be null.
 
 [![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)
 
-
+# PS:Ocelot currently does not support 2.0
```

Comment:
This [wiki ](https://github.com/TomPallister/Ocelot/wiki/Service-discovery)page is where we put information about how to run consul with Ocelot. If you want to edit this information let me know.